tests/issues/issue-635.php

```diff
@@ -4033,11 +4033,11 @@
             'isModified' => false,
             'CssVersion' => '3',
             'Device_Name' => 'N1',
-            'Device_Maker' => 'Nokia',
-            'Device_Type' => 'Tablet',
-            'Device_Pointing_Method' => 'touchscreen',
+            'Device_Maker' => 'TizzBird',
```

Comment:
This one seems like it could go several different ways, why did you ultimately decide on this device?

http://useragent.mkf.solutions/?userAgent=Mozilla%2F5.0+%28Linux%3B+Android+4.0.4%3B+N1+Build%2FIMM76D%29+AppleWebKit%2F537.36+%28KHTML%2C+like+Gecko%29+Chrome%2F38.0.2125.114+Mobile+Safari%2F537.36

Other possibilities based on that analysis from other parsers (outdated now):

Newman N1
Oppo N1

Reply:
Nokia N1: Android 5.0 ... 5.1
Newman N1: Android 4.4
Oppo N1: Android 4.2
TizzBird N1: Android 4.0

I decided based on the android version.

Reply:
Ah, nice, good call